plugin/fulltext/mysql-test/ngram/ngram_token_size_basic.test

```diff
@@ -0,0 +1,50 @@
+
+#
+# show the global and session values;
+#
+call mtr.add_suppression("option 'ngram-token-size': signed value 0");
+call mtr.add_suppression("option 'ngram-token-size': signed value -1");
+call mtr.add_suppression("option 'ngram-token-size': signed value 11");
+
+select @@global.ngram_token_size;
+--error ER_INCORRECT_GLOBAL_LOCAL_VAR
+select @@session.ngram_token_size;
+show global variables like 'ngram_token_size';
+show session variables like 'ngram_token_size';
+--disable_warnings
+select * from information_schema.global_variables where variable_name='ngram_token_size';
+select * from information_schema.session_variables where variable_name='ngram_token_size';
+--enable_warnings
+
+# show it's read-only
+--error ER_INCORRECT_GLOBAL_LOCAL_VAR
+set global ngram_token_size = 3;
+
+# Valid values are from 1 to 10
+# ngram_token_size = 1 if we try to set ngram_token_size <= 0
+--let $restart_parameters= --ngram_token_size=0
+--source include/restart_mysqld.inc
+--disable_warnings
+select * from information_schema.global_variables where variable_name='ngram_token_size';
+--enable_warnings
```

Comment:
Why disable warnings?

Reply:
Tests were rewritten, and all these statements went away.

However there is still a place where warnings are disabled. It's in the test that inserts CJK sample texts into tables with various charsets. For example, Shift-JIS doesn't support Korean, so inserting Korean text will generate warnings. Those are expected and just pollute the result log. These warnings are suppressed with `disable_warnings`.